en-us/Content/Topics/ReleaseNotes/access-gateway-release-status.htm

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <meta name="robots" content="noindex, nofollow" />
+        <meta http-equiv="REFRESH" content="0.0;url=https://help.okta.com/oag/en-us/Content/Topics/ReleaseNotes/access-gateway-release-notes.htm" />

```

Comment:
I see we have an alias for this `ext_oag_releasenotes`, should we use it instead of the full url? (I only know that because of your other PR)

Reply:
In this case, we only have EN content for OAG; if we go with an alias, then we'd have to explicitly set the locale to EN, and that would mess with any JA users who might possibly land (well, pass through) here, because it'd change their cookie value.

Reply:
Got it.... we are good then, we will need to remember somehow that this link needs to change when we have JA content for OAG. Can you file a jira and put it in the backlog please?